package.json

```diff
@@ -1,61 +1,63 @@
 {
-  "name": "@smg-automotive/example",
+  "name": "@smg-automotive/eslint-config",
   "version": "0.0.0-development",
   "description": "A boilerplate package setup",
-  "main": "dist/cjs/index.js",
-  "module": "dist/esm/index.js",
-  "files": [
-    "dist/**/*"
-  ],
   "publishConfig": {
     "access": "public"
   },
   "scripts": {
     "version": "npm run build",
-    "build": "rimraf dist && rollup -c",
-    "lint": "eslint --ext ts,js,tsx,jsx,json .",
+    "build": "rimraf dist && tsc -p tsconfig.build.json --outDir dist",
```

Comment:
Could we still use rollup here? I know there isn't much to build and you're essentially using rollup to run `tsc`, but it gives us consistency and less mental overhead when switching projects.

Reply:
I've never tried to use RollUp for compiling TS to JS only. But for the consistency, it's worth, let me update the build mechanism